modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -200,6 +200,7 @@ account-entry-email-domains-configuration-name=Email Domains
 account-entry-group-configuration-description=Define which types of accounts are allowed on this site.
 account-entry-group-configuration-name=Accounts
 account-entry-restricted-object-field-id=Account Restricted Field
+account-entry-x-not-has-postal-address-y=The account (accountId={0}) does not have that postal address (postalAddressesId={1})
```

Comment:
Looks like this is a new pattern and likely to get rejected. I suggest replacing it with:
`Postal address {1} not found for account {0}.`

Reply:
Changed! ([commit](https://github.com/liferay-appsec/liferay-portal/pull/1415/commits/a87fc1578657157f6846f11c4bfba22bc300bed8))